tests/kernel/smp/boards/qemu_cortex_a53_smp.overlay

```diff
@@ -0,0 +1,19 @@
+/* Copyright 2022 Carlo Caione <ccaione@baylibre.com>
```

Comment:
Not for sure, do you want to set the `qemu_cortext_a53_smp ` board 4 cores as default? Since the 4 or 8 cores platform is very common I think. Or are there some limitations?

Reply:
I'll do that in a different PR so that we can spot if that causes any problem.